.github/workflows/pr.yml

```diff
@@ -24,4 +24,24 @@ jobs:
       - name: Run coding style check
         run: make code-style-check
       - name: Run tests
-        run: make tests
\ No newline at end of file
+        run: make tests
+
+  oauth_checks:
+    name: oauth checks
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: oauth
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Pull docker-compose
+        run: cp -n docker-compose.yml.dist docker-compose.yml && docker-compose -f docker-compose.yml pull
+      - uses: satackey/action-docker-layer-caching@v0.0.11
+        continue-on-error: true
+      - name: Prepare containers
+        run: make start && make composer-install
+      - name: Run coding style check
+        run: make code-style-check
+      - name: Run tests
```

Comment:
You can comment the lines for run tests so you don't have to install phpunit in this PR